packages/esm-patient-chart-app/src/config-schema.ts

```diff
@@ -152,6 +152,11 @@ export const esmPatientChartSchema = {
     _description: 'Whether to require an active visit for the encounter tile',
     _default: true,
   },
+  enableEmbeddedEncounterView: {
+    _type: Type.Boolean,
+    _description: "When enabled, the encounter summary will be displayed using the Form Engine's embedded view mode",
+    _default: true,
```

Comment:
```suggestion
    _default: false,
```

Don't we want this to be `false` by default so the old behaviour is preserved and then implementers can opt-in to the new functionality by toggling the config property to `true`?

Reply:
Actually if we do want to have it as a "system default", it makes sense to set it to `true`. For now, I've set it to `false`, but we can revisit that decision later.